authentication/securestore/main.go

```diff
@@ -64,11 +64,14 @@ func IsBackendAvailable(backend string) (isAvailable bool) {
 }
 
 // AvailableBackends provides a slice of all available backend keys on the current OS.
-//
-// Note: CGO_ENABLED=1 is required for darwin builds (enables OSX Keychain)
 func AvailableBackends() []string {
 	b := []string{}
 
+	if runtime.GOOS == "darwin" {
+		// Assume Keychain is always available on Darwin. It will not return from keyring.AvailableBackends()
+		b = append(b, "keychain")
```

Comment:
is this because keychain is built into the OS? How far back does that go in Mac history? just making sure we dont need to do a more sophisticated check and thow some kind of "not supported" exception on older macs. Or check explicitly the app/version installed...

Its not a huge concern, just trying to error-proof if its simple enough to do.

Reply:
Yep exactly. [First release was 1999](https://en.wikipedia.org/wiki/Keychain_(software)). 

Unofficially, if for some reason the version of macOS does not contain keychain, errors will be handled during CRUD operations.

Officially, our CLIs officially should only support latest OS releases and this should be a non-issue.